src/types/frozenset.rs

```diff
@@ -157,6 +162,35 @@ mod impl_ {
 
 pub use impl_::*;
 
+#[inline]
+pub(crate) fn new_from_iter<T: ToPyObject>(
+    py: Python<'_>,
+    elements: impl IntoIterator<Item = T>,
+) -> PyResult<Py<PyFrozenSet>> {
+    #[inline]
+    fn new_from_iter_inner(
```

Comment:
Why is the inner function necessary?

Reply:
To avoid generic code bloat, this inner part is not generic - actually will quickly just force-push to remove `#[inline]` from this as it's a bulkier loop and unlikely to benefit much from inlining.

Reply:
Got it, I always forget about that aspect.